OTMonsterConverter/Converter/TfsXmlConverter.cs

```diff
@@ -382,8 +382,10 @@ private void xmlToGeneric(TFSXmlMonster tfsMonster, out Monster monster)
                     monster.Summons.Add(new Summon()
                     {
                         Name = summon.name,
-                        Chance = (double)summon.chance / 100,
-                        Rate = (uint)((summon.interval > 0) ? summon.interval : summon.speed)
+                        Chance = (uint)Math.Min(100, summon.chance),
```

Comment:
`Monster` class should store percents in percentage format. How about `Chance = Math.min(1, (double)summon.chance / 100)`